src/client/dfs/dfs.c

```diff
@@ -307,8 +285,8 @@ dfs_oclass_select(daos_handle_t poh, daos_oclass_id_t oc_id,
 static int
 oid_gen(dfs_t *dfs, daos_oclass_id_t oclass, bool file, daos_obj_id_t *oid)
 {
-	daos_ofeat_t	feat = 0;
-	int		rc;
+	enum daos_otype_t type = 0;
```

Comment:
I'd suggest to use DAOS_OT_MULTI_HASHED explicitly

Reply:
sure